Add note on the benchmarks (#1180)
withastro/astro.build: src/content/blog/astro-4120.mdx

### Performance

> <Note>[partialprerendering.com](https://www.partialprerendering.com/) has had some changes since this post was published. Rerunning the benchmarks maybe produce different results. We stand by these results as accurate the time of publishing.</Note>

Reviewer: ```suggestion
<Note>[partialprerendering.com](https://www.partialprerendering.com/) has had some changes since this post was published. Rerunning the benchmarks now may produce different results. These results were accurate at the time of publishing.</Note>
```

Author: oops, sorry i pushed before i saw this.

I think either is fine, no preference from me.

Reviewer: Whichever you prefer is fine, but fix the maybe!
